src/kindlemint/agents/base_agent.py

```
@@ -189,7 +191,8 @@ async def send_message(self, message: AgentMessage) -> bool:
             if self.agent_registry:
                 return await self.agent_registry.route_message(message)
             else:
-                self.logger.warning("No agent registry available for message routing")
+                self.logger.warning(
+                    "No agent registry available for message routing")
                 return False
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)